doc/utils/manifest.md

```diff
@@ -24,6 +24,10 @@ devices = ["nanos", "nanox", "nanos+", "stax"]
 [tests]
 unit_directory = "./unit-tests/"
 pytest_directory = "./tests/"
+
+[build_options]
+debug = "DEBUG=1"
+test = "DEBUG=1"
 ```
 
 ### Sections
```

Comment:
Could we clearly separate `Required sections` and `Optional sections`  each into its own markdown section ? ^^ @lpascal-ledger what do you think ?

Reply:
I don't think there should be any `Required sections`. Let's just use and suggest a common naming and not add more mandatory / enforced stuff in the manifest